### ISSUE TITLE
Inline Internal Targets render as anchor macros without their Text

### ISSUE THREAD

**Issue by RobertSeeger** [sphinx-contrib/confluencebuilder#630]
Inline Internal Targets, as described in https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#inline-internal-targets, are rendered correctly with the help of the Confluence `anchor` macro (implicitly transforming the text to an anchor id for the macro parameter by lower casing it), however the text of the target is not emitted at all.

This has been noticed in particular in definition lists, where the definition term only consists of an inline internal target - in this case the definition list in Confluence will be visually distorted, as the <dt> tag does not contain any visual text. But it can also be seen within a regular paragraph text.

Example RST source:

```
Test Inline Internal Targets
============================

_`Term One Target`
 Definition Text for Term One
 
Term Two - No Target
 Definition Text for _`Term Two` has inline internal target
 
 
This is a paragraph with an _`Example Inline Target`.
```

Rendered in HTML (partial excerpt)

```
  <section id="test-inline-internal-targets">
<h1>Test Inline Internal Targets<a class="headerlink" href="#test-inline-internal-targets" title="Permalink to this headline"></a></h1>
<dl class="simple">
<dt><span class="target" id="term-one-target">Term One Target</span></dt><dd><p>Definition Text for Term One</p>
</dd>
<dt>Term Two - No Target</dt><dd><p>Definition Text for <span class="target" id="term-two">Term Two</span> has inline internal target</p>
</dd>
</dl>
<p>This is a paragraph with an <span class="target" id="example-inline-target">Example Inline Target</span>.</p>
</section>
```

Rendered in CONF (partial excerpt)

```
<dl>
<dt><ac:structured-macro ac:name="anchor">
<ac:parameter ac:name="">term-one-target</ac:parameter>
</ac:structured-macro>
</dt>
<dd>
<p>Definition Text for Term One</p>
</dd>
<dt>Term Two - No Target</dt>
<dd>
<p>Definition Text for <ac:structured-macro ac:name="anchor">
<ac:parameter ac:name="">term-two</ac:parameter>
</ac:structured-macro>
 has inline internal target</p>
</dd>
</dl>
<p>This is a paragraph with an <ac:structured-macro ac:name="anchor">
<ac:parameter ac:name="">example-inline-target</ac:parameter>
</ac:structured-macro>
.</p>

```

As you can see, the texts

- Term One Target
- Term Two
- Example Inline Target

have completely disappeared in the CONF output.

**Comment by RobertSeeger** [comment]
Environment:

docutils                        0.17.1
Sphinx                          4.4.0
sphinx-epytext                  0.0.4
sphinx-rtd-theme                1.0.0
sphinxcontrib-applehelp         1.0.2
sphinxcontrib-confluencebuilder 1.7.1
sphinxcontrib-devhelp           1.0.2
sphinxcontrib-htmlhelp          2.0.0
sphinxcontrib-jsmath            1.0.1
sphinxcontrib-qthelp            1.0.3
sphinxcontrib-serializinghtml   1.1.5

**Comment by jdknight** [comment]
Thanks for reporting this issue. Changes have been added in #631 to support inline internal targets (to properly include their text). This change has been added into the main branch and should be made available next stable release (v1.8). Users needing to use this feature before a release is made, the [development version](https://sphinxcontrib-confluencebuilder.readthedocs.io/install/#development-installation) of this extension can be installed. If trying out this correction still yields issues, please add a comment here.

**Comment by jdknight** [comment]
v1.8 is now available on PyPI -- marking as closed.
